test/test_atacac/test_validators/test_builtin.py

```diff
@@ -0,0 +1,115 @@
+import os
+import re
+import textwrap
+from test.utils import MockDir
+
+import pytest  # pylint: disable=import-error
+import yamale
+from yamale.validators import DefaultValidators
+
+from atacac.validators.builtin import Example, Dict
+
+
+class TestExample:
```

Comment:
Why class? All other tests are just functions

Reply:
Already mentioned in https://github.com/europ/atacac/pull/11#discussion_r454222298